webui/react/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": ">=16.13",
+    "node": ">=16.13 <17",
```

Comment:
I don't really know if this is the correct syntax or not but the idea lgtm :)

Reply:
I wasn't sure either so I looked this up